app/models/user.rb

```diff
@@ -8,7 +8,7 @@ class User < CouchRest::Model::Base
   property :password_salt, String, :accessible => true
   property :contact_email, String, :accessible => true
   property :contact_email_key, String, :accessible => true
-
+  property :invite_code, String, :accessible => true
```

Comment:
@elijh this is the only open question for me right now. Do we need to store the invite code used?